test/code/plugins/in_npmd_server_plugintest.rb

```diff
@@ -28,10 +28,10 @@ class NPMDServerTest < Test::Unit::TestCase
 
     CMD_ENUMERATE_BINARY_INSTANCES = "ps aux | grep fake_npmd"
 
-    CMD_START_NPMD  = "StartNPMD"
-    CMD_STOP_NPMD   = "StopNPMD"
-    CMD_CONFIG_NPMD = "ConfigNPMD"
-    CMD_PURGE_NPMD  = "PurgeNPMD"
+    CMD_START_NPMD  = "StartNPM"
```

Comment:
if they should be the same as the one defined in the in_npmd_server.rb, please keep 1 copy.